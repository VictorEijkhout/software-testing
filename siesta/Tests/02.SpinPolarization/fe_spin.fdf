%include ../base_fe3.fdf
SystemLabel         fe_spin

Spin polarized

