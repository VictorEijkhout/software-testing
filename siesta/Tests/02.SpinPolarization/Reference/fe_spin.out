Executable      : /home/fnpedron/Siesta/siesta-dev/_build_serial/Src/siesta
Version         : 5.2.0-15-g8c00451d2
Architecture    : x86_64
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch -O3 -march=native
Environment variables: 
  SIESTA_PS_PATH: /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos
Parallelisations: none
NetCDF support
NetCDF-4 support
Lua support
DFT-D3 support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/_build_serial/Tests/02.SpinPolarization/fe_spin/work
* Running in serial mode.
>> Start of run:  17-JAN-2025  12:34:56

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

 NOTE: Siesta V5 introduces some new defaults for basis-set generation
 NOTE: which might lead to slightly different numerical results.
 NOTE: Please see the manual for full details and compatibility measures.


reinit: Reading from file ../../fe_spin.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: Linear Fe3 cluster
reinit: -----------------------------------------------------------------------
reinit: System Label: fe_spin
reinit: -----------------------------------------------------------------------

NOTE on units for spin output: 
{S} is the spin magnetic moment, in units of the Bohr magneton,
assuming a g-factor for the electron spin of exactly 2.
The spin magnetic moment is then numerically equal to the charge inbalance,
in units of electrons, between the spin-up and -down channels.


initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   26 Label: Fe_nc

Ground state valence configuration (from tables):   4s02  4p00  3d06  4f00

---- Processing specs for species: Fe_nc
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Fe_nc.psf

---- Pseudopotential check for Fe

Pseudized shells:
4s( 1.00) rc: 2.00
4p( 0.00) rc: 2.00
3d( 7.00) rc: 2.00
4f( 0.00) rc: 2.00
Valence configuration for ps generation: (assumed as above)
For Fe_nc, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Fe_nc                Z=  26    Mass=  55.845        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=4
          i=1  nzeta=1  polorb=1  (4s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=4
          i=1  nzeta=0  polorb=0  (4p)  (perturbative polarization orbital) (from 4s)
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Fe_nc                 (Z =  26)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    8.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  2.8472
V l=1 = -2*Zval/r beyond r=  2.8472
V l=2 = -2*Zval/r beyond r=  2.8472
V l=3 = -2*Zval/r beyond r=  2.8472
All V_l potentials equal beyond r=  1.9894
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.8472
Using large-core scheme (fit) for Vlocal

atom: Estimated core radius    2.84716
Fit of Vlocal with continuous 2nd derivative
Fitting vlocal at       1.9954
Choosing vlocal chloc cutoff:  5741  3.100644
qtot up to nchloc:    8.00000259
atom: Maximum radius for chlocal:    3.10064
atom: Maximum radius for r*vlocal+2*Zval:    3.10064

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.991366   el= -0.319855   Ekb=  3.826230   kbcos=  0.253712
GHOST: No ghost state for L =  1
   l= 1   rc=  2.019582   el= -0.070931   Ekb=  3.335986   kbcos=  0.155791
GHOST: No ghost state for L =  2
   l= 2   rc=  1.989365   el= -0.307264   Ekb=-12.632066   kbcos= -0.657060
GHOST: No ghost state for L =  3
   l= 3   rc=  2.015527   el=  0.019538   Ekb= -1.701666   kbcos= -0.006384

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    7.355669
                 energy =   -0.308198
                kinetic =    0.284874
    potential(screened) =   -0.593072
       potential(ionic) =   -5.763042

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.584362
                 energy =   -0.296412
                kinetic =    8.773647
    potential(screened) =   -9.070058
       potential(ionic) =  -18.006145

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 4s
Split based on tail norm

   izeta = 1
                     rc =    7.355669
                 energy =    0.004656
                kinetic =    0.554633
    potential(screened) =   -0.549977
       potential(ionic) =   -5.337881
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 4s( 2.00)                                                            
 4p( 0.00)                                                            
 3d( 6.00)                                                            
Vna: chval, zval:    8.00000   8.00000

Vna:  Cut-off radius for the neutral-atom potential:   7.355669
comcore: Pseudo-core radius Rcore=  3.839063

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   26 Fe_nc                   # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Fe_nc                 2                    # Species label, number of l-shells
 n=4   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   7.356   
   1.000   
 n=3   2   1                         # n, l, Nzeta 
   4.584   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Fe_nc.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                            (in Bohr units)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   3.64400  1        1
siesta:      0.00000   0.00000   0.00000  1        2
siesta:      0.00000   0.00000  -3.64400  1        3

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      3    27    48

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = collinear
redata: Number of spin components                   = 2
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken charges (when)               = geometry+end
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Write Hirshfeld charges (when)              = none
redata: Write Voronoi charges (when)                = none
redata: Write spin charge (when)                    = scf
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   100.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       20
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Antiferro initial spin density              =   F
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   300.0000 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 8
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
mix.SCF: Spin-component mixing                   all
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 8
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    24.000000
Total ionic charge:    24.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        6.350127    0.000000    0.000000
        0.000000    6.350127    0.000000
        0.000000    0.000000   10.583544

outcell: Cell vector modules (Ang)   :    6.350127    6.350127   10.583544
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    426.7720
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.0000 nnzs=729, refcount: 7>
  <dData1D:(new from dSpData1D) n=729, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...

initDM: WARNING: noncollinear spins in DM.InitSpin not used because nspin < 4
DM filled with atomic data (user-defined):
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.0000 nnzs=729, refcount: 8>
  <dData2D:DM n=729 m=2, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

InitMesh: MESH = 40 x 40 x 64 = 102400
InitMesh: Mesh cutoff (required, used) =   100.000   101.065 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       236.750053
siesta: Eions   =      2102.576618
siesta: Ena     =        92.089899
siesta: Ekin    =      2809.864634
siesta: Enl     =     -1814.901850
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =      -117.576648
siesta: DUscf   =        56.508730
siesta: DUext   =         0.000000
siesta: Ex      =     -1118.003241
siesta: Ec      =       -97.852654
siesta: Exc     =     -1215.855896
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -2220.460126
siesta: Etot    =     -2292.447748
siesta: FreeEng =     -2292.486337

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1    -2220.460126    -2292.447748    -2292.486337  0.993786 -9.427434 35.275907
     spin moment: {S} , |S| = {        0.0        0.0    6.00000 }     6.00000
timer: Routine,Calls,Time,% = IterSCF        1       0.030   9.25
   scf:    2    -2296.906084    -2294.724148    -2294.761584  0.023437 -9.297429 33.557682
     spin moment: {S} , |S| = {        0.0        0.0    6.00000 }     6.00000
   scf:    3    -1865.897130    -2186.561378    -2186.570827  0.972547 -6.078310 43.730081
     spin moment: {S} , |S| = {        0.0        0.0    2.00000 }     2.00000
   scf:    4    -1134.895809    -1819.225438    -1819.299709  1.539477 15.732926172.761986
     spin moment: {S} , |S| = {        0.0        0.0    1.17116 }     1.17116
   scf:    5    -1875.619989    -1964.226429    -1964.226429  1.175272 -2.403483 91.339800
     spin moment: {S} , |S| = {        0.0        0.0    0.00000 }     0.00000
   scf:    6    -1906.630458    -1935.875102    -1935.875102  0.064250 -1.892985 98.032616
     spin moment: {S} , |S| = {        0.0        0.0    0.00000 }     0.00000
   scf:    7    -1973.464545    -1954.992223    -1954.992223  0.075059 -2.921026 92.295440
     spin moment: {S} , |S| = {        0.0        0.0    0.00000 }     0.00000
   scf:    8     -823.515552    -1993.726326    -1993.777846  0.997079  1.554984 61.032611
     spin moment: {S} , |S| = {        0.0        0.0    4.00000 }     4.00000
   scf:    9    -1650.028304    -1829.212254    -1829.214826  0.573317  3.100989 77.215100
     spin moment: {S} , |S| = {        0.0        0.0    0.01615 }     0.01615
   scf:   10     -476.437582    -1919.239239    -1919.239239  0.999335 -3.931004 95.243294
     spin moment: {S} , |S| = {        0.0        0.0   -0.00000 }    -0.00000
   scf:   11       47.958809    -1762.791365    -1762.810131  1.171317  5.396844 94.453104
     spin moment: {S} , |S| = {        0.0        0.0    0.19436 }     0.19436
   scf:   12    -1021.916316    -2080.461885    -2080.510101  1.532109 -8.180973 57.350336
     spin moment: {S} , |S| = {        0.0        0.0    2.00000 }     2.00000
   scf:   13      -45.953373    -1758.579133    -1758.633816  1.413672  4.405469 90.320494
     spin moment: {S} , |S| = {        0.0        0.0    1.91862 }     1.91862
   scf:   14    -1763.132712    -1760.878794    -1760.918893  0.147889  4.732729 91.550406
     spin moment: {S} , |S| = {        0.0        0.0    1.94044 }     1.94044
   scf:   15    -1770.456568    -1765.673594    -1765.711398  0.103184  4.259534 88.571627
     spin moment: {S} , |S| = {        0.0        0.0    1.97653 }     1.97653
   scf:   16    -1844.160397    -1805.296131    -1805.333477  0.366495  2.355450 75.760703
     spin moment: {S} , |S| = {        0.0        0.0    1.97905 }     1.97905
   scf:   17    -1740.190550    -1773.163081    -1773.203696  0.345901  2.733439 80.491592
     spin moment: {S} , |S| = {        0.0        0.0    1.97966 }     1.97966
   scf:   18    -1781.999705    -1777.649949    -1777.660848  0.113617  2.154486 77.329574
     spin moment: {S} , |S| = {        0.0        0.0    1.99641 }     1.99641
   scf:   19    -1769.910001    -1773.942114    -1773.990403  0.228613  2.072195 78.920254
     spin moment: {S} , |S| = {        0.0        0.0    2.00000 }     2.00000
   scf:   20    -1750.227696    -1762.158665    -1762.204832  0.149630  2.736869 81.533365
     spin moment: {S} , |S| = {        0.0        0.0    2.00000 }     2.00000
SCF_NOT_CONV: SCF did not converge  in maximum number of steps.
Geom step, scf iteration, dDmax, dHmax:    0    20      0.149630     5.992592

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

siesta: E_KS(eV) =            -1762.1587

siesta: E_KS - E_eggbox =     -1762.1587

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000    0.000000   -0.000000
----------------------------------------
   Max   50.806439
   Res   23.950385    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max   50.806439    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -38.49      -38.49      748.95       -0.00        0.00       -0.00
(Free)E + p*V (eV/cell)    -1821.8685
Target enthalpy (eV/cell)    -1762.2048

mulliken: Atomic and Orbital Populations:

mulliken: Spin UP 

Species: Fe_nc               
Atom  Qatom  Qorb
               4s      3dxy    3dyz    3dz2    3dxz    3dx2-y2 4Ppy    4Ppz    
               4Ppx    
   1  2.740   1.121  -0.000   0.009   0.030   0.009  -0.000   0.464   0.644
              0.464
   2  7.519   0.177   1.000   1.003   1.014   1.003   1.000   1.051   0.220
              1.051
   3  2.740   1.121  -0.000   0.009   0.030   0.009  -0.000   0.464   0.644
              0.464

mulliken: Qtot =       13.000

mulliken: Spin DOWN 

Species: Fe_nc               
Atom  Qatom  Qorb
               4s      3dxy    3dyz    3dz2    3dxz    3dx2-y2 4Ppy    4Ppz    
               4Ppx    
   1  1.729   1.034  -0.000   0.007   0.022   0.007  -0.000   0.078   0.503
              0.078
   2  7.543   0.231   1.000   1.009   1.014   1.009   1.000   1.058   0.163
              1.058
   3  1.729   1.034  -0.000   0.007   0.022   0.007  -0.000   0.078   0.503
              0.078

mulliken: Qtot =       11.000

Mulliken Atomic Populations:
Atom #   charge [q] valence [e]      Sz [e]  Species
     1     3.530838    4.469162    1.011735  Fe_nc
     2    -7.061676   15.061676   -0.023470  Fe_nc
     3     3.530838    4.469162    1.011735  Fe_nc
-------------------------------------------
 Total    -0.000000                2.000000

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       397.850477
siesta: Eions   =      2102.576618
siesta: Ena     =        92.089899
siesta: Ekin    =      1330.750883
siesta: Enl     =      -738.083404
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       164.635193
siesta: DUscf   =       500.358182
siesta: DUext   =         0.000000
siesta: Ex      =      -920.022222
siesta: Ec      =       -89.310578
siesta: Exc     =     -1009.332800
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -1750.227696
siesta: Etot    =     -1762.158665
siesta: FreeEng =     -1762.204832

siesta: Final energy (eV):
siesta:  Band Struct. =     397.850477
siesta:       Kinetic =    1330.750883
siesta:       Hartree =    1382.923045
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -920.022222
siesta:         Corr. =     -89.310578
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =   -1009.332800
siesta:  Ion-electron =   -3274.812413
siesta:       Ion-ion =    -191.687379
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =   -1762.158665
siesta:         Fermi =       2.736869

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.000000    0.000000  -50.806439
siesta:      2   -0.000000    0.000000    0.000002
siesta:      3    0.000000    0.000000   50.806437
siesta: ----------------------------------------
siesta:    Tot   -0.000000    0.000000   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.024025   -0.000000    0.000000
siesta:    -0.000000   -0.024025    0.000000
siesta:     0.000000   -0.000000    0.467456

siesta: Cell volume =        426.771969 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:          -0.00152264          0.00014420  Ry/Bohr**3
siesta:          -0.13980231          0.01323986  eV/Ang**3
siesta:        -223.98799657         21.21258635  kBar
(Free)E+ p_basis*V_orbitals  =       -1760.130627
(Free)Eharris+ p_basis*V_orbitals  =       -1748.199659

mulliken: Atomic and Orbital Populations:

mulliken: Spin UP 

Species: Fe_nc               
Atom  Qatom  Qorb
               4s      3dxy    3dyz    3dz2    3dxz    3dx2-y2 4Ppy    4Ppz    
               4Ppx    
   1  2.740   1.121  -0.000   0.009   0.030   0.009  -0.000   0.464   0.644
              0.464
   2  7.519   0.177   1.000   1.003   1.014   1.003   1.000   1.051   0.220
              1.051
   3  2.740   1.121  -0.000   0.009   0.030   0.009  -0.000   0.464   0.644
              0.464

mulliken: Qtot =       13.000

mulliken: Spin DOWN 

Species: Fe_nc               
Atom  Qatom  Qorb
               4s      3dxy    3dyz    3dz2    3dxz    3dx2-y2 4Ppy    4Ppz    
               4Ppx    
   1  1.729   1.034  -0.000   0.007   0.022   0.007  -0.000   0.078   0.503
              0.078
   2  7.543   0.231   1.000   1.009   1.014   1.009   1.000   1.058   0.163
              1.058
   3  1.729   1.034  -0.000   0.007   0.022   0.007  -0.000   0.078   0.503
              0.078

mulliken: Qtot =       11.000

Mulliken Atomic Populations:
Atom #   charge [q] valence [e]      Sz [e]  Species
     1     3.530838    4.469162    1.011735  Fe_nc
     2    -7.061676   15.061676   -0.023470  Fe_nc
     3     3.530838    4.469162    1.011735  Fe_nc
-------------------------------------------
 Total    -0.000000                2.000000

cite: Please see "fe_spin.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.2.0-15-g8c00451d2
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.043
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.006       1.006   100.00
Setup                  1       0.042       0.042     4.13
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       0.964       0.964    95.83
geom_init              1       0.257       0.257    25.51
state_init             1       0.024       0.024     2.38
hsparse                1       0.000       0.000     0.01
overlap                1       0.023       0.023     2.33
Setup_H0               1       0.203       0.203    20.20
naefs                  2       0.000       0.000     0.01
dnaefs                 2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.011       0.023     2.25
nlefsm                 2       0.066       0.133    13.20
DHSCF_Init             1       0.049       0.049     4.83
DHSCF1                 1       0.006       0.006     0.58
InitMesh               1       0.006       0.006     0.58
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.043       0.043     4.24
REMESH                 1       0.003       0.003     0.32
REORD                106       0.000       0.008     0.77
PHION                  1       0.030       0.030     2.98
COMM_BSC             180       0.000       0.010     1.01
POISON                23       0.002       0.049     4.89
fft                   46       0.001       0.037     3.67
setup_H               21       0.030       0.628    62.41
DHSCF                 22       0.033       0.724    71.96
DHSCF3                22       0.030       0.656    65.21
rhoofd                22       0.007       0.146    14.53
XC                    22       0.014       0.304    30.22
GXC-CellXC            22       0.013       0.282    28.01
gridxc@cellXC         22       0.013       0.282    28.00
vmat                  22       0.005       0.113    11.19
IterSCF               20       0.030       0.608    60.38
compute_dm            20       0.000       0.006     0.57
diagon                20       0.000       0.006     0.55
r-eigvec              40       0.000       0.004     0.43
r-buildHS             40       0.000       0.000     0.01
rdiag                 40       0.000       0.004     0.42
rdiag1                40       0.000       0.000     0.04
rdiag2                40       0.000       0.001     0.09
rdiag3                40       0.000       0.003     0.25
rdiag4                40       0.000       0.000     0.02
r-buildD              20       0.000       0.001     0.09
MIXER                 20       0.000       0.001     0.10
PostSCF                1       0.099       0.099     9.87
FINAL_HF               1       0.099       0.099     9.87
DHSCF4                 1       0.068       0.068     6.75
dfscf                  1       0.059       0.059     5.84
overfsm                1       0.000       0.000     0.03
state_analysis         1       0.000       0.000     0.03
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.02
optical                1       0.000       0.000     0.00
  

>> End of run:  17-JAN-2025  12:34:57
Job completed
